docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -497,6 +497,23 @@ And finally, run it with:
 docker run -i --rm -p 8080:8080 quarkus-quickstart/getting-started
 ----
 
+=== Manually using the minimal base image
```

Comment:
According to this file[1] content looks differently. Is this expected?

[1] https://github.com/quarkusio/quarkus/pull/22617/files#diff-64db9d9387ea534bab41b295ce8f3fb0ade553d8b5a21a7c174c01c21b001ca7

Reply:
Could you be more explicit? I have no idea what you're complaining about :).

Reply:
Dockerfile from `docs/src/main/asciidoc/building-native-image.adoc` (lines 506-512):
```
FROM <...>
WORKDIR /work/
COPY target/*-runner /work/application
RUN chmod 775 /work
EXPOSE 8080
CMD <...>
```
Dockerfile from `docs/src/main/asciidoc/quarkus-runtime-base-image.adoc` (lines 115-125):
```
FROM <...>
WORKDIR /work/
RUN chown 1001 /work \
    && chmod "g+rwX" /work \
    && chown 1001:root /work
COPY --chown=1001:root target/*-runner /work/application
EXPOSE 8080
USER 1001
CMD <...>
```

It looks like these commands target the same case, yet they are different. What I am curious about, is if it intentional, or not?